qp_spades/qiita-plugin-ci.yml

```diff
@@ -0,0 +1,162 @@
+# name: Qiita Plugin CI
```

Comment:
I think this file needs to be deleted. The only one that should exist should be in .github/workflows not in qp_spades right?